homeassistant/components/tellduslive/entry.py

```diff
@@ -111,3 +111,15 @@ def _last_updated(self):
     def unique_id(self) -> str:
         """Return a unique ID."""
         return self._id
+
+    @property
+    def device_info(self):
+        """Return device info."""
+        device = self._client.device_info(self.device.device_id)
+        return {
+            'identifiers': {('tellduslive', self.device.device_id)},
+            'name': self.device.name,
+            'model': device['model'].title(),
+            'manufacturer': device['protocol'].title(),
+            'via_hub': ('tellduslive', device.get('client')),
```

Comment:
Does this id match the hub id on line 93 in the component?

Reply:
Yes, for the Cloud devices. Device-info will not show the hub when connecting to a local hub.